packages/create-llama/e2e/utils.ts

```diff
@@ -96,7 +96,7 @@ export function runCreateLlama(
     "--model",
     MODEL,
     "--open-ai-key",
-    "testKey",
+    process.env.OPENAI_API_KEY || "testKey",
```

Comment:
@thucpn 

In the job, we're still getting `testKey` (https://github.com/run-llama/LlamaIndexTS/actions/runs/7246220748/job/19737708679) - might it be that `test` is starting a new process and this child process doesn't have the right `process.env`?

Reply:
not the case as it works here: https://github.com/thucpn/LlamaIndexTS/actions/runs/7245927631/job/19736878992